src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.Unix.cs

```diff
@@ -19,140 +19,135 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal class NegoState
+    internal partial class NegoState
     {
-        internal const int MaxReadFrameSize = 64 * 1024;
-        internal const int MaxWriteDataSize = 63 * 1024; // 1k for the framing and trailer that is always less as per SSPI.
-
-        internal NegoState(Stream innerStream, bool leaveStreamOpen)
+        internal IIdentity GetIdentity()
         {
             throw new PlatformNotSupportedException();
         }
 
-        internal static string DefaultPackage
+        internal static string QueryContextAssociatedName(SafeDeleteContext securityContext)
         {
-            get
-            {
-                throw new PlatformNotSupportedException();
-            }
+            throw new PlatformNotSupportedException();
```

Comment:
We implemented all of these.. In the successive PR https://github.com/dotnet/corefx/pull/5773  you can see.. so this is kind of temporary .